docs/mautic_community_portal/signing_in_and_configuring_your_profile.rst

```diff
@@ -0,0 +1,251 @@
+
+.. vale on
+
+Accept the terms and conditions
+*******************************
+
+.. vale off
+
+The first time you log in - and any time thereafter when there have been substantial changes to the terms and conditions - you must accept them before you proceed. To do this, review the terms and conditions and, at the bottom of the page, click 'I agree with these terms'. You can't continue using the platform if you refuse the terms, and a popup gives you the option to delete your account if you wish.
+
+..vale on
+
+.. image:: images/accept-terms-and-conditions.png
+    :alt: VSCode screenshot showing how to change branches
+    :width: 600px
+    :align: center
+
+Certain cookies permit features on the portal which you can choose to accept or reject on the cookie popup. It's important to understand that certain features such as embedded note taking won't work if you don't accept the relevant cookies. A prompt notifies you of this if you come across resources like this, to review and update your settings to use that feature if your current setting restricts it.
+
+.. image:: images/cookie-consent.png
+    :alt: VSCode screenshot showing how to change branches
+    :width: 600px
+    :align: center
+
+.. raw:: html
+
+    <br><br>
+
+Setting language preferences
+============================
+
+.. vale off
+
+At the top of the page, you'll notice that there are different languages available if you prefer. Select the language you wish to use and the interface changes into that language. It's worth noting that some aspects of the website may not be fully translated because they're community created, and as a result may only be available in the language they were originally created in. If you would like to help with translating more of the content please post in #translations on :xref:`slack`.
+
+.. vale on
+
+Claiming membership
+*******************
+Once logged in you have the option to claim a Mautic membership.
+
+There is a :xref:`membership-option` as a way to support the Mautic community while also being able to join in with governance processes like voting. You can become an individual member for $100 a year or a pro rated amount based on where you pay taxes, and you can become a corporate member. Read more about the :xref:`membership-option` and explore the perks of corporate membership in this :xref:`corporate-membership`.
+
+If you are already a member of Mautic, you can select this option to match up your membership with your account on the community portal. This allows you to participate in votes, engage in the General Assembly, create proposals, and so forth.
+
+You can watch more about claiming your membership status in this short video:
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/9kOa759Y4eQ" frameborder="0" allowfullscreen></iframe>
+
+.. raw:: html
```

Comment:
Is this necessary?

Reply:
This syntax is added right before a <br> tag, so it adds a space between the image and the text.